src/pages/optimized-for-change/index.zh-hans.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 应变而优
+date: '2018-12-12'
+spoiler: 如何写好一个API？
+---
+
+如何写好一个API？
+
+*优秀* 的 API即好记住又明确，促进代码可读性、准确性和高效性，帮助开发者走向[成功](https://blog.codinghorror.com/falling-into-the-pit-of-success/)。
+
+我称这些设计方向为“第一阶层”，因为它们是类库开发者需要关注的第一件事情。你可能不得不舍弃其中一部分并作出权衡，但至少它们在你脑海中游走过。
```

Comment:
```suggestion
我称这些设计方向为「第一阶层」，因为它们是类库开发者需要关注的第一件事情。你可能不得不舍弃其中一部分并作出权衡，但至少它们在你脑海中游走过。
```
I suggest using square brackets to make it more readable. See references [here](https://github.com/mzlogin/chinese-copywriting-guidelines#%E5%85%A8%E8%A7%92%E5%92%8C%E5%8D%8A%E8%A7%92).